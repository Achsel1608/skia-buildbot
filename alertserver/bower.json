{
  "name": "alerts",
  "version": "0.1.0",
  "authors": [],
  "description": "Alert Server",
  "license": "BSD",
  "private": true,
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "third_party/bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "polymer": "Polymer/polymer#^1.0.0",
    "iron-elements": "PolymerElements/iron-elements#~1.0.0",
    "paper-elements": "PolymerElements/paper-elements#~1.0.0"
  },
  "resolutions": {
    "polymer": "^1.1.0",
    "marked-element": "^1.0.0",
    "web-animations-js": "2.1.2",
    "webcomponentsjs": "^0.7.2"
  }
}
