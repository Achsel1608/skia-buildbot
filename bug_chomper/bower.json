{
  "name": "bug_chomper",
  "version": "0.1.0",
  "authors": [],
  "description": "Tool for quickly prioritizing bugs.",
  "license": "BSD",
  "private": true,
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "third_party/bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "polymer": "Polymer/polymer#~0.5.1",
    "core-elements": "Polymer/core-elements#~0.5.1",
    "paper-elements": "Polymer/paper-elements#~0.5.1",
    "webcomponentsjs": "Polymer/webcomponentsjs#^0.5.1"
  },
  "resolutions": {
    "WeakMap": "~0.2.1",
    "webcomponentsjs": "^0.6.0"
  }
}
