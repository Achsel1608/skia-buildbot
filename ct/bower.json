{
  "name": "ctfe",
  "version": "0.1.0",
  "authors": [],
  "description": "Cluster Telemetry Frontend",
  "license": "BSD",
  "private": true,
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "third_party/bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "polymer": "Polymer/polymer#^0.9.0",
    "iron-elements": "PolymerElements/iron-elements#~0.9.1",
    "paper-elements": "PolymerElements/paper-elements#~0.9.1",
    "layout": "Polymer/layout"
  },
  "resolutions": {
    "polymer": "^0.9.0"
  }
}
