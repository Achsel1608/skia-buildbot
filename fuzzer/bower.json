{
  "name": "fuzzer",
  "version": "0.0.0",
  "authors": [
    "Greg Humphreys <humper@google.com>"
  ],
  "license": "MIT",
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "polymer": "Polymer/polymer#~0.5.4",
    "paper-elements": "Polymer/paper-elements#~0.5.4",
    "core-elements": "Polymer/core-elements#~0.5.4",
    "webcomponents": "Polymer/webcomponents",
    "webcomponentsjs": "~0.5.4"
  }
}
