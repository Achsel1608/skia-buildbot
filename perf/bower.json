{
  "name": "perf",
  "version": "0.1.0",
  "authors": [],
  "description": "skiaperf.com",
  "license": "BSD",
  "private": true,
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "third_party/bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "polymer": "Polymer/polymer#^1.0.3",
    "iron-elements": "PolymerElements/iron-elements#^1.0.0",
    "paper-elements": "PolymerElements/paper-elements#^1.0.0",
    "d3": "~3.5.6",
    "wcstamp": "~1.0.0"
  }
}
