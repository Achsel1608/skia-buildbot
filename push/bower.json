{
  "name": "push",
  "version": "0.1.0",
  "authors": [],
  "description": "skiapush.com",
  "license": "BSD",
  "private": true,
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "third_party/bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "native-promise-only": "~0.7.6-a",
    "polymer": "Polymer/polymer#^1.0.2",
    "iron-elements": "PolymerElements/iron-elements#^1.0.0",
    "paper-elements": "PolymerElements/paper-elements#^1.0.0"
  },
  "resolutions": {
    "polymer": "^1.0.2"
  }
}
