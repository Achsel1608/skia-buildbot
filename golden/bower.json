{
  "name": "golden",
  "version": "0.1.0",
  "authors": [],
  "description": "Rebaseline Tool",
  "license": "BSD",
  "private": true,
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "third_party/bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "polymer": "Polymer/polymer#~0.5.4",
    "core-elements": "Polymer/core-elements#~0.5.4",
    "paper-elements": "Polymer/paper-elements#~0.5.4",
    "webcomponentsjs": "Polymer/webcomponentsjs#^0.5.4",
    "flot": "~0.8.3",
    "d3": "~3.5.6"
  },
  "resolutions": {
    "webcomponentsjs": "^0.6.0"
  }
}
